textmap/tokenizers.py

```diff
@@ -1,17 +1,26 @@
 from .utilities import flatten
+from warnings import warn
+
 from sklearn.base import BaseEstimator, TransformerMixin
-from nltk.tokenize import sent_tokenize, word_tokenize, MWETokenizer
+from nltk.tokenize import sent_tokenize, word_tokenize, MWETokenizer, TweetTokenizer
 from nltk.collocations import BigramCollocationFinder
 from nltk.metrics import BigramAssocMeasures
 from sklearn.feature_extraction.text import CountVectorizer
 
-import spacy.lang.en
-import stanza
+# Optional tokenization packages
+try:
+    import stanza
+except ImportError:
+    warn("The stanza library could not be imported StanzaTokenizer will not be available.")
+try:
+    import spacy
+except:
```

Comment:
```suggestion
except ImportError:
```
We don't want to accidentally catch some other exception.